darwin/cli_functions.py

```diff
@@ -20,33 +20,34 @@
 from darwin.utils import find_files, persist_client_configuration, secure_continue_request
 
 
-def authenticate(email: str, password: str, projects_dir: str) -> Config:
+def authenticate(api_key: str, datasets_dir: str, default_team: bool) -> Config:
     """Authenticate user against the server and creates a configuration file for it
 
     Parameters
     ----------
-    email : str
-        Email to use for the client login
-    password : str
-        Password to use for the client login
-    projects_dir : str
+    team : str
+        Teams to use for the client login
+    api_key : str
+        API key to use for the client login
+    datasets_dir : str
          String where the client should be initialized from
 
     Returns
     -------
     Config
     A configuration object to handle YAML files
     """
-    # Resolve the home folder if the project_dir starts with ~ or ~user
-    projects_dir = Path(os.path.expanduser(projects_dir))
-    Path(projects_dir).mkdir(parents=True, exist_ok=True)
-    print(f"Projects directory created {projects_dir}")
+    # Resolve the home folder if the dataset_dir starts with ~ or ~user
+    datasets_dir = Path(os.path.expanduser(datasets_dir))
+    Path(datasets_dir).mkdir(parents=True, exist_ok=True)
+    print(f"Datasets directory created {datasets_dir}")
 
     try:
-        client = Client.login(email=email, password=password, projects_dir=projects_dir)
+        client = Client.login(api_key=api_key, datasets_dir=datasets_dir)
         config_path = Path.home() / ".darwin" / "config.yaml"
         config_path.parent.mkdir(exist_ok=True)
-        return persist_client_configuration(client)
+        default_team = client.team if default_team else None
+        return persist_client_configuration(client, default_team=default_team)
 
     except InvalidLogin:
         _error("Invalid credentials")
```

Comment:
Should the error message hint to the API key or we leave a generic "credentials" ?